swan-cern/scripts/before-notebook.d/10_spark.sh

```diff
@@ -22,10 +22,13 @@ then
   _log "Globally enabling the Spark extensions"
   jq -n --argjson sparkconnector/extension true \
         '{load_extensions: $ARGS.named}' > /etc/jupyter/nbconfig/notebook.json
+
+  sed -i "1s/^/c.InteractiveShellApp.extensions.append('sparkconnector.connector')\n/" \
+  /home/$NB_USER/.ipython/profile_default/ipython_kernel_config.py
```

Comment:
can you put a tab here?

Reply:
Can you also test that the code above is necessary?

Reply:
I added a tab and tested the code mentioned. From what I see it is not required.

Reply:
What is the code that is not required? the jq command that injects in notebook.json?

Reply:
Yes